R/stage_calc_ts.R

```diff
@@ -2,7 +2,7 @@
 #' @description accept calculated/modeled data as a data.frame and return a file handle
 #' 
 #' @param sites a character vector of valid NWIS site IDs
-#' @param variable short name of variable \code{\link{get_ts_variables}}
+#' @param var short name of variable as in \code{get_var_codes(out='var')}
```

Comment:
I like this language. `var` is the short names we are using for the `variables`. Makes a lot of sense :+1: